site/en/install/_index.yaml

```diff
@@ -39,7 +39,7 @@ landing_page:
     - heading: Download a package
       description: >
         <p>Install TensorFlow with Python's <em>pip</em> package manager.</p>
-        <aside class="note">TensorFlow 2 packages require a <code>pip</code> version >19.0.</aside>
+        <aside class="note">TensorFlow 2 packages require a <code>pip</code> version >20.3 when running on macOS 11, or >19.0 on all other systems.</aside>
```

Comment:
Technically it should be >=20.3, as 20.3 is sufficient

(I wonder whether this is also true for the ">19.0")

Reply:
Yeah, this is indicated on [pip-specific page](https://github.com/tensorflow/docs/blob/master/site/en/install/pip.html#L34) and should follow here. Ideally, that's the page with the more exhaustive requirements and this provides the "high level" for folks that don't click through